src/components/Popover/PopoverContent.tsx

```diff
@@ -30,14 +30,22 @@ export interface PopoverContentProps {
     | 'right-center'
     | 'right-top';
 
+  /**
+   * Whether the popover should hide when a click outside it occurs
+   */
+  closeOnOutsideClicks?: boolean;
+
   /**
    * @ignore
    */
   children: ReactNode;
 }
 
 const PopoverContent = React.forwardRef<HTMLDivElement, PopoverContentProps>(
-  function PopoverContent({ alignment = 'bottom-left', children, ...rest }, forwardedRef) {
+  function PopoverContent(
+    { alignment = 'bottom-left', children, closeOnOutsideClicks = true, ...rest },
```

Comment:
`[nit / suggestion]` Having the default value for this prop be `true` can lead to a less ergonomic interface, we may be able to tighten it up with a slight rename, flipping the logic, defaulting the new prop to `false`

```
// Right now the only time you use the prop, you HAVE to put the value
<PopoverContent closeOnOutsideClick={false} />

// If we change the name, whenever you use the boolean prop all you gotta do
// is declare it
<PopoverContent ignoreOutsideClicks />

// Default value as `false`
<PopoverContent />
```

Reply:
Yeah I thought about it, since I have the same approach when it comes to adding props (default should be `false` to help with readability & expectations), but I couldn't find a name that made sense. `ignoreOutsideClicks` is barely ok since it doesn't fully clarify what "ignore" means.

I'll go ahead and use it though :)

Reply:
Renamed to `persistOnOutsideClicks` :)